game/islandtrolltribes/scripts/vscripts/itt.lua

```diff
@@ -482,22 +482,79 @@ function ITT:AdjustSkills( hero )
-        {"npc_creep_wolf_jungle", {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_hide_wolf", 100}, {"item_bone", 100}},
-        {"npc_creep_bear_jungle", {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_hide_jungle_bear", 100}, {"item_bone", 100}},
-        {"npc_creep_bear_jungle_adult", {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_hide_jungle_bear", 100}, {"item_bone", 100}},
-        {"npc_creep_panther", {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_bone", 100}, {"item_bone", 100}},
-        {"npc_creep_panther_elder", {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_bone", 100}, {"item_bone", 100}},
-        {"npc_creep_lizard", {"item_meat_raw", 100}},
-        {"npc_creep_fish", {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}},
-        {"npc_creep_fish_green", {"item_meat_raw", 100}},
-        {"npc_creep_hawk", {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_bone", 100}, {"item_egg_hawk", 10}},
-        {"npc_creep_mammoth", {"item_bone", 100},{"item_bone", 100},{"item_bone", 100},{"item_bone", 100},{"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_meat_raw", 100}, {"item_horn_mammoth", 100}, {"item_horn_mammoth", 50}}
+        {"npc_creep_elk_wild", {"item_hide_elk", 100}, {"item_bone", 100}},
+        {"npc_creep_wolf_jungle", {"item_hide_wolf", 100}, {"item_bone", 100}},
+        {"npc_creep_bear_jungle", {"item_hide_jungle_bear", 100}, {"item_bone", 100}},
+        {"npc_creep_bear_jungle_adult", {"item_hide_jungle_bear", 100}, {"item_bone", 100}},
+        {"npc_creep_panther", {"item_bone", 100}, {"item_bone", 100}},
+        {"npc_creep_panther_elder", {"item_bone", 100}, {"item_bone", 100}},
+        {"npc_creep_hawk", {"item_bone", 100}, {"item_egg_hawk", 10}},
+        {"npc_creep_mammoth", {"item_bone", 100},{"item_bone", 100},{"item_bone", 100},{"item_bone", 100}, {"item_horn_mammoth", 100}, {"item_horn_mammoth", 50}}
     }
+    local meatTable = {
```

Comment:
Noticeably missing from the drops: baby forms of animals, adult forms of animals besides jungle bear, and the secret boss.